transformer_engine/jax/praxis/transformer.py

```diff
@@ -148,6 +148,8 @@ class TransformerLayer(TransformerEngineBaseLayer):
     float32_attention_logits: bool = False
     layer_type: TransformerLayerType = TransformerLayerType.ENCODER
     self_attn_mask_type: str = 'causal'
+    enable_rope: bool = False
```

Comment:
We should document or link to some rope documentation somewhere.

Reply:
Done